module/account/src/Application/Controller/Api/Account/AvatarChangeAction.php

```diff
@@ -74,7 +74,7 @@ public function __construct(CommandBusInterface $commandBus, FormFactoryInterfac
      *     description="Success"
      * )
      *
-     * @ParamConverter(class="Ergonode\Account\Domain\Entity\User")
+     * @ParamConverter(name="user")
```

Comment:
It looks to me like you can use the name value from the configuration inside the Converter instead of adding annotations every time you use it.